ibis/expr/api.py

```diff
@@ -463,6 +463,23 @@ def group_concat(arg, sep=',', where=None):
     return _ops.GroupConcat(arg, sep, where).to_expr()
 
 
+def arbitrary(arg, where=None, how='first'):
+    """
+    Selects the first encountered value.
```

Comment:
This should probably read something like: "Selects the first non-null value in a column" so that users know what happens when `NULL` values are encountered.